src/wiseagents/agents/utility_wise_agents.py

```diff
@@ -45,7 +46,7 @@ def __repr__(self):
             destination_agent_name={self.destination_agent_name},\
             response_delivery={self.response_delivery}"
      
-    def process_request(self, request):
+    def handle_request(self, request):
```

Comment:
A general question here: if I well understand use this handle_* methods is up to the agent implementation. Maybe we should document them underlying it's important to use them. For example you haven't reviewed the new "ChatAgent" (it's fine I can take this), but can you confirm me it would make sense to use these utility methods also there to have a similar way to do things?
This is a question and not preventing to approve and merge the PR, which I'm going to move on

Reply:
Typically, agent implementations just need to implement the `process_request` method and the default `handle_request` method implementation will take care of everything else depending on the type of collaboration in use (i.e., sequential or phased).

In this case, I had initially updated the `PassThroughClientAgent` to just implement `handle_request` since it isn't involved in collaboration but I can actually just update it to implement `process_request` instead because we do have handling in `handle_request` for the case where sequential and phased collaboration are not being used.

For the `ChatAgent`, it should implement `handle_request` because it needs to make use of the chat history even though phased collaboration is not in use.

I'll submit a PR with these changes shortly.